docker-compose.yml

```
@@ -14,7 +14,7 @@ services:
 
     mongodb:
         container_name: mongodb
-        image: mongodb/mongodb-atlas-local:latest
+        image: mongodb/mongodb-atlas-local:8.0.8
```

We can safely use the latest of version 8 instead of locking to this fix version.

```suggestion
        image: mongodb/mongodb-atlas-local:8
```